clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```
@@ -45,7 +42,7 @@
   },
   "ADD CORRESPONDING": {
     "prefix": "ADD CORRESPONDING",
-    "body": ["ADD CORR ${1:id} TO ${2:id}"],
+    "body": ["ADD ${1|CORR,CORRESPONDING|} ${2:id} TO ${3:id}"],
```

This just looks confusing to me - offering two synonyms + one option (the empty one) that does not fit the description of the snippet.
```suggestion
    "body": ["ADD CORRESPONDING ${1:id} TO ${2:id}"],
```